server/resourcemanager/resource_manager.go

```diff
@@ -129,29 +143,41 @@ func (m *manager[T]) RegisterRoutes(r *mux.Router) *mux.Router {
 
 	enabledOps := m.EnabledOperations()
 
+	listHandler := m.methodNotAllowed
```

Comment:
we used to not register unsupported routes. This caused that requests that should be handled by resourcemanager went directly to the SPA handler, making the response be the main `index.html` contents from the web directory.

Now, we always register all routes, but return a nice `405 method not allowed` for disabled methods